### ISSUE TITLE
provide locale to getTimeZones()

### ISSUE THREAD

**Issue by ofbeaton** [vvo/tzdb#78]
It would be nice to be able to provide the locale to getTimeZones() so that we can get the timezone strings in the language of the user. Right now it is hardcoded to English (and a specific one) which means this library cannot be used to populate tz pickers when displaying a UI in another language.

https://github.com/vvo/tzdb/blob/master/lib/getTimeZones.js

**Comment by vvo** [comment]
@ofbeaton Good idea on the paper, not sure how if it would work given the current code so your best bet is to try it by forking this repository and then let me know.

I am interested in this feature yes and hope you can find a way to implement it!

Let me know :)

**Comment by ofbeaton** [comment]
I mean a start might be to allow the user to pass in the locale in a backwards compatible way...

```
export default function getTimeZones(locale = "en-US") {
```

But obviously more work would need to be done... Sorry if this is too obvious.

**Comment by vvo** [comment]
@ofbeaton Yep that would work as a start, again I am up for the feature but I won't implement it myself as I do not need it at the moment. But you're welcome to try it and open a PR if you see it working well.

Thanks!

**Comment by MohamedHegab** [comment]
Any luck @ofbeaton as I need the same issue right now
